### ISSUE TITLE
Define how hub talks with clients over AMQP

### ISSUE THREAD

**Issue by rexxars** [vgno/ssehub#7]
Suggestion: JSON. Open to alternatives if it makes hub faster or less complicated.

Example:

``` json
{
    "path": "/some/path",
    "event": "optional event name",
    "id": "optional event id",
    "retry": 3500,
    "data": "some arbitrary data"
}
```

Event, id and retry are optional. Data and path are required.
If ID is not set, the hub skips adding the event to the history.

Thoughts?

**Comment by olesku** [comment]
Let's stick with this format, except that we also support to get the target channel name from the amqp routingkey if the "id" field is not present in the JSON data.

**Comment by olesku** [comment]
Implemented in bf33f3c62162a9c8046ec9a24dd8e7934bfdc97d
